### DISCUSSION TITLE
Lightning preset and SDXL lightning models

### DISCUSSION THREAD

**Discussion by wisdombox44** [lllyasviel/Fooocus#2782] [Q&A] [answered]
Is lightning preset and performance works with only SDXL models. I tried with juggernautLightning and DreamshaperLightning models. I got burn out and deformed results.
is there a preset for SDXL Lightning models.
What is your suggestion. Thank you

**Comment by poor7** [comment, marked as answer]
There is no such preset. Use the parameters recommended by the developers of these models, which can usually be found in the model description on the Civitai website.

**Comment by wisdombox44** [comment]
Is that mean ligtning preset only developed for Normal SDXL models. For ligtning models we need to play with settings and make a New preset. That would be nice if we have 2 different lightning preset by default
Thank You.

**Reply by mashb1t** [reply]
The lightning preset has been created with maximum usability in mind, so people can just quickly change models and do not have to download multiple ones depending on the selected performance, which would be inconvenient.
